toolkit/scripts/toolchain.mk

```diff
@@ -297,3 +305,23 @@ $(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(depend_REBUILD_TOOLCHAIN)
 		$(call print_error,\nToolchain download failed. See above errors for more details.) \
 	}
 endif
+
+# ./out/RPMS is reserved for RPMs generated by the tooling, all other RPMs are cached in the ./build folder. If REBUILD_TOOLCHAIN=y is set
+# the toolchain RPMs should also be placed into out/RPMs.
+
+# ./out/SRPMS are still handled by build_official_toolchain_rpms.sh as a side effect of building the toolchain tar.gz.
+ifeq ($(REBUILD_TOOLCHAIN),y)
+$(RPMS_DIR): $(toolchain_out_rpms)
+
+# For each toolchain rpm in ./out/RPMS, add a dependency on its counterpart in the normal toolchain directory
+$(foreach toolchain_out_rpm,$(toolchain_out_rpms),$(eval $(toolchain_out_rpm): $(filter %/$(notdir $(toolchain_out_rpm)),$(toolchain_rpms))))
+$(toolchain_out_rpms): ; $(warning $^)
+	@if [ ! -f "$@" \
+			-o "$^" -nt "$@" ] ; then \
+		echo "Placing built toolchain RPM $(notdir $@) into $(RPMS_DIR)" && \
+		cp $^ $@; \
+	fi || $(call print_error, Failed to duplicate '$^' to '$@' )
+else
+$(toolchain_out_rpms):
+	@$(call print_error, Cannot build toolchain rpm ($@) unless REBUILD_TOOLCHAIN=y is set)
```

Comment:
Do we need the `else` part? In other words: is it even possible for any target to require `toolchain_out_rpms` if `REBUILD_TOOLCHAIN != y`? Unless the user manually tries to `make [toolchain_rpm]` but that's not something they should do and expect to work.

Reply:
Probably not needed, but I wanted a bit of hint on what might be happening if things go wrong so I can debug more easily.

Reply:
Git it. I'd still remove it in the final version. I think we shouldn't try to safeguard against any possible misuse of `make`. We'd probably end up with piles of warning and code that's never used.

Reply:
Sure, I'll drop it.